.gitignore

```diff
@@ -16,6 +16,7 @@ lerna-debug.log*
 # Tests
 /coverage
 /.nyc_output
+/redisinsight/api/test/test-runs/coverage
```

Comment:
is it better to use `coverage/` or `**/coverage` instead of full path's?

Reply:
thanks, changed to `**/coverage`